docs/config-app.md

```diff
@@ -23,10 +23,12 @@ This parameter affects how many CPU cores will be utilized by the application. R
 - `http-client.connect-timeout-ms` - set the connect timeout.
 
 ## Auction
+- `auction.update-rates-period` - default default period to update currency rates.
 - `auction.default-timeout-ms` - default operation timeout for OpenRTB Auction requests.
 - `auction.max-request-size` - set the maximum size in bytes of OpenRTB Auction request.
 - `auction.stored-requests-timeout-ms` - timeout for stored requests fetching.
 - `auction.expected-cache-time-ms` - approximate value in milliseconds for Cache Service interacting. 
+- `ad-server-currency` - default currency for auction, if its value was not specified in request
```

Comment:
One more thing: it's not stated anywhere what value this parameter accepts. In PBS-Go PR there is a comment: "// Currency strings must be ISO 4217 alpha codes". If that is the case here let's mention it in documentation and validate the code when the server starts up.